osquery/requests/serializers/json.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include "osquery/requests/requests.h"
+
+namespace osquery {
+
+/**
+ * @brief JSON Serializer
+ */
+class JSONSerializer : public Serializer {
```

Comment:
There's a lot of JSON action within the codebase. Maybe this can go somewhere more accessible?